### PR TITLE
Add env to frontend config.

### PR DESCRIPTION
**Pull request by logan-anderson**
This will allow  one to populate env vars on the Frontend that are not populated with TINA_PUBLIC or NEXT_PUBLIC

https://github.com/tinacms/tinacms/pull/4298 must be merged first

### CODE REVIEW

File: packages/@tinacms/cli/src/next/vite/index.ts [hunk 1]
```diff
@@ -140,6 +141,22 @@ export const createConfig = async ({
...
+      // if the value is a string, we can just use it
+      if (typeof tinaEnv[key] === 'string') {
+        publicEnv[key] = tinaEnv[key] as string
+      } else {
+        // otherwise, we need to stringify it
+        publicEnv[key] = JSON.stringify(tinaEnv[key])
+      }
+    } catch (error) {
+      // if we can't stringify it, we'll just warn the user
+      console.warn(`Could not stringify public env tinaEnv.${key} env variable`)
```
[the remark this answers is not in the document]
Comment (logan-anderson) [reply packages/@tinacms/cli/src/next/vite/index.ts]: Updated the error message.

[the remark this answers is not in the document]
Comment (kldavis4) [reply packages/@tinacms/cli/src/next/vite/index.ts]: I still don't know why the user would know what tinaEnv means - is it mentioned somewhere else?

[the remark this answers is not in the document]
Comment (logan-anderson) [reply packages/@tinacms/cli/src/next/vite/index.ts]: I removed tinaEnv from the error message. It now says "Could not stringify env.${key} env from tina/config file'

File: packages/tinacms-authjs/src/tinacms.ts [hunk 2]
```diff
@@ -73,7 +73,11 @@ export class UsernamePasswordAuthJSProvider extends DefaultAuthJSProvider {
   async authenticate(props: { username: string; password: string }) {
     const csrfToken = await getCsrfToken()
     // TODO make api baseUrl configurable
```
Comment (kldavis4) [inline packages/tinacms-authjs/src/tinacms.ts:75]:
```suggestion
```

**Comment by logan-anderson** [comment]
@kldavis4 This doesn't solve the next auth issues but do we still want to merge it? (It could be a useful in down the road)

**Comment by kldavis4** [comment]
> @kldavis4 This doesn't solve the next auth issues but do we still want to merge it? (It could be a useful in down the road)

It seems like it could be useful but may want to check in with @jeffsee55

**Comment by logan-anderson** [comment]
We may want to bring this back at some point but I am going to close for now.

**Pull request closed without merging**